### ISSUE TITLE
WordPress on Azure Plug-in issue - Application initialization failed, please check your app.json file: Http failure response for http://cascorp.azurewebsites.net/wp-content/plugins/bullhorn-oscp/app.json: 404 Not Found

### ISSUE THREAD

**Issue by fsailer** [bullhorn/career-portal#364]
INSERT SHORT DESCRIPTION EXPLAINING THE HIGH-LEVEL REASON FOR THE NEW ISSUE HERE.

## Current behavior
The plugin installed and activated without issue. I entered the Bullhorn provided values for Stream and Corp Token, however when adding the [oscp] to a new page the following error is shown: Application initialization failed, please check your app.json file: Http failure response for http://cascorp.azurewebsites.net/wp-content/plugins/bullhorn-oscp/app.json: 404 Not Found.

The Bullhorn rep was not able to point me to the appropriate resolution for this, however indicated that it was probably related to the json file being blocked by the web host. Unfortunately we are the provider and host of this MS Azure WordPress instance and require more technical assistance.
-

## Expected behavior
Plugin renders
-

## Screenshots

![image](https://user-images.githubusercontent.com/11092343/99421184-1bd02f80-28cc-11eb-9b7d-2181dacedb7f.png)

__Current__

__Expected__

**Comment by pbearne** [comment]
Hi 
This does sound like the webserver doesn't have .json as file format look in the server configs to see the allowed file type

If you get stuck you could use Matador Jobs WordPress plugin https://matadorjobs.com/ as this doesn't have json config files

Paul

**Comment by fsailer** [comment]
For reference - This WP instance is WordPress for Windows hosted on our Microsoft Azure instance. The version is 5.4.4 with an option to upgrade to 5.5.3. The underlying web server is therefore IIS.

I can confirm all the files are in the ../wwwroot/sp-content/plugins/bullhorn-oscp/ directory including the .json file with our companies swimlane and corp key. A line in the web.config file was added for .json files but this has made no difference. Additionally the "Upload File Types" plug-in was added to manage whitelist file types and .json was added with a MIME type of "application/json" and this has made no difference.

The underlying MS Azure Web App hosting the WP instance does not have a management tool for limiting/allowing specific file types so I am left believing this is specific to the WP instance. What else can I check or what troubleshooting tools beyond the typical browser console can I use to help determine where this problem is occurring?

**Comment by charlesabarnes** [comment]
When I navigate to the portal now, it appears as if it loads and no error is returned.  Now I imagine it may be a caching thing on your end.  Below is a screenshot of the portal.

![image](https://user-images.githubusercontent.com/4327754/99531853-fba27e00-2968-11eb-9f85-86a826d4f02e.png)


Please let me know if you have any more questions
